pages/Book-Binder.md

```diff
@@ -0,0 +1,19 @@
+The Book Binder is an [Electric Machine](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) which can be used to combine different books together (much like an anvil would).  
+
+## Book Binder Settings
+By default, you cannot use this machine to get books above the vanilla limit.  
+However for server owners it is possible to adjust this and either add a custom limit, or no limit at all.
+
+| Name                     | Type    | Default Value |
+| ------------------------ | ------- | ------------- |
+| bypass-vanilla-max-level | boolean | false         |
+| has-custom-max-level     | boolean | false         |
+| custom-max-level         | int     | 15            |
+
+**Tip:** Updating machine specific settings can be done in items.yml
+
+## Book Binder Tiers
```

Comment:
I'd probably have tiers above settings. That's what players will care about and players outnumber server owners

Reply:
fair point